executor/aggfuncs/func_json_objectagg_test.go

```diff
@@ -143,9 +143,11 @@ func (s *testSuite) TestMemJsonObjectagg(c *C) {
 			}
 		}
 
-		aggTest := buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, false)
+		aggTest1 := buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, false)
+		tests = append(tests, aggTest1)
+		aggTest2 := buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, true)
+		tests = append(tests, aggTest2)
```

Comment:
```
tests := []aggTest{
buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, true),
buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, false),
}
```

Reply:
Do you mean `aggTest ` -> `multiArgsAggMemTest`?
```
tests := []multiArgsAggMemTest{
buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, true),
buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, false),
}
```
The test case generated by the following for loop is greater than 2,  `tests = append(tests, aggTest1)` may be better?
```
for k := 0; k < len(argCombines); k++ {

}
```

Reply:
yes, I mean that
```
tests := []multiArgsAggMemTest{
    buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, true),
    buildMultiArgsAggMemTester(ast.AggFuncJsonObjectAgg, argTypes, mysql.TypeJSON, numRows, aggfuncs.DefPartialResult4JsonObjectAgg, defaultMultiArgsMemDeltaGens, false),
}
```

In this case only have 2 test cases, using above code is better?

Reply:
We can use multi loop like follow for generating the test cases with multi arg.
https://github.com/pingcap/tidb/blob/c704b9756e20d172da32e95f2c7bec3c40dbaff3/executor/aggfuncs/func_max_min_test.go#L309

Reply:
@mmyj PTAL~

Reply:
I think almost is ok.